chacha8rand.md

```diff
@@ -0,0 +1,85 @@
+# ChaCha8Rand
+
+ChaCha8Rand is a ChaCha8-based key-erasure CSPRNG with performance similar to non-cryptographic random number generators, designed to be [the default source for the `math/rand{,v2}` and `runtime` Go packages][].
```

Comment:
Does key-erasure need to be defined or a link to a definition added?

Reply:
Opened #46.